library/Zend/Amf/Server.php

```diff
@@ -709,8 +722,9 @@ public function getRequest()
     /**
      * Public access method to private Zend_Amf_Server_Response reference
      *
-     * @param  string|Zend_Amf_Server_Response $response
+     * @param string|Zend_Amf_Server_Response $response
```

Comment:
Zend_Amf_Server_Response class does not exist:
- https://github.com/zf1s/zf1/pull/137/files